<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGEGL3DViewportForm</class>
 <widget class="QWidget" name="FGEGL3DViewportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget#widget_cont{
	border : 1px solid;
	border-color : black;
	border-radius : 3px;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_cont" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">.QWidget#widget{
	padding : 4px;
	background-color: rgb(0, 0, 0);
}
</string>
        </property>
        <widget class="QWidget" name="topwidget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>542</width>
           <height>28</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <widget class="QPushButton" name="pushButton_widget_class">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/select.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_splite_widget">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    background-repeat: no-repeat;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/splitter.svg</normaloff>:/Icons/svg/splitter.svg</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_mode_widget">
          <property name="geometry">
           <rect>
            <x>258</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    background-repeat: no-repeat;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/splitter.svg</normaloff>:/Icons/svg/splitter.svg</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_menu_add_mesh">
          <property name="geometry">
           <rect>
            <x>286</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    background-repeat: no-repeat;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/add_mesh.svg</normaloff>:/Icons/svg/add_mesh.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_selection_models">
          <property name="geometry">
           <rect>
            <x>314</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    background-repeat: no-repeat;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/tweak1_selection.svg</normaloff>:/Icons/svg/tweak1_selection.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_type_transform">
          <property name="geometry">
           <rect>
            <x>342</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/global7_gizmo.svg</normaloff>:/Icons/svg/global7_gizmo.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_display_mode">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/solid_display_mode.svg</normaloff>:/Icons/svg/solid_display_mode.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_object_interaction_mode">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>398</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/solid_display_mode.svg</normaloff>:/Icons/svg/solid_display_mode.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_armature">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>426</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
	background-color: rgb(40, 120, 240);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/blender_icons/con_armature.svg</normaloff>:/Icons/blender_icons/con_armature.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>456</x>
            <y>0</y>
            <width>27</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_projection_mode">
          <property name="geometry">
           <rect>
            <x>484</x>
            <y>0</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/Icons/svg/projection_perspective.svg</normaloff>:/Icons/svg/projection_perspective.svg</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="autoRepeatDelay">
           <number>300</number>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QWidget" name="widget_menu" native="true">
          <property name="geometry">
           <rect>
            <x>28</x>
            <y>0</y>
            <width>202</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>202</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_menu {
      background-color: white;
  }
QMenuBar {
	  color: rgb(0, 0, 0);
      background-color: white;
      margin: 0px; /* some spacing around the menu */
      spacing: 3px; /* spacing between menu bar items */
      min-height: 28px; 
}
</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="FGEOpenGLWidget" name="glwidget_world" native="true">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
QMenu {
	  color: rgb(0, 0, 0);
      border-radius: 3px;
      background-color: white;
      margin: 2px; /* some spacing around the menu */
  }

  QMenu::item {
	height : 30px;
      padding: 2px 25px 2px 20px;
      border: 1px solid transparent; /* reserve space for selection border */
  }

  QMenu::item:selected {
		border-radius: 3px;
		background-color: rgb(235, 235, 235);
  }</string>
        </property>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>30</width>
           <height>58</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">.QWidget{
	border-radius:2px;
	border : 1px solid black;
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <widget class="QPushButton" name="pushButton_snap">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>29</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/snap.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 40, 40);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_selection">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>1</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/select.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 40, 40);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>30</width>
           <height>117</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">.QWidget{
	border-radius:2px;
	border : 1px solid black;
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <widget class="QPushButton" name="pushButton_transformation">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>88</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/transform.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 40, 40);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_scale">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>59</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/scale.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 40, 40);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_rotation">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>30</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/rotate.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 40, 40);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_translation">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>1</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton{
	border : 0px solid;
    border-image: url(&quot;:/Icons/translate.png&quot;);
    background-repeat: no-repeat;
	background-color: rgb(40, 40, 40);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FGEOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>3DViewport/Primitive/FGEOpenGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
